dataland-qa-service/src/main/kotlin/org/dataland/datalandqaservice/services/QaReportMetadataService.kt

```diff
@@ -25,7 +25,7 @@ class QaReportMetadataService(
     /**
      * Method to search all data and the connected meta information associated with a data set.
      * @param uploaderUserIds set of user ids of the uploader
-     * @param showOnlyActive whether to show only active data
+     * @param showOnlyActive whether to show only active qa reports
```

Comment:
Erscheint mir sinnvoll aber wieso ist dieser change in diesem PR?

Reply:
s.o.